src/app/clusters/scenes/SceneTable.h

```diff
@@ -0,0 +1,363 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+#pragma once
+
+#include <app-common/zap-generated/cluster-objects.h>
+#include <app/clusters/scenes/ExtensionFieldSetsImpl.h>
+#include <lib/support/CHIPMemString.h>
+#include <lib/support/CommonIterator.h>
+#include <lib/support/CommonPersistentData.h>
+#include <lib/support/Span.h>
+
+namespace chip {
+namespace scenes {
+
+enum SceneTLVTag
+{
+    kTagSceneStorageIDContainer = 1,
+    kTagSceneEndpointID,
+    kTagSceneGroupID,
+    kTagSceneID,
+    kTagSceneDataContainer,
+    kTagSceneName,
+    kTagSceneDTransitionTime,
+    kTagSceneDTransitionTime100,
+};
+
+typedef uint32_t TransitionTimeMs;
+typedef uint16_t SceneTransitionTime;
+typedef uint8_t TransitionTime100ms;
+
+static constexpr uint8_t kEFSBufferSize     = 128;

```

Comment:
You have `CHIP_CONFIG_SCENES_MAX_EXTENSION_FIELDSET_SIZE_PER_CLUSTER` elsewhere. Why is that here?

This should be moved to SceneTableImpl.cpp. It is an implementation detail, not an interface detail.

Reply:
removed in latest commit.